payments/models.py

```diff
@@ -57,52 +66,102 @@ def external_account(self, bank_account):
 
 class StripeEvent(models.Model):
     event_id = models.CharField(primary_key=True, max_length=100, blank=True)
+    user_id = models.CharField(max_length=100, blank=True)
     type = models.CharField(max_length=100, blank=True)
     message_text = models.TextField()
+    verified = models.BooleanField(default=False)
     processed = models.BooleanField(default=False)
+    created = models.DateTimeField(null=True, blank=True)
```

Comment:
Can we change this (and other `DateTimeField`s to use `auto_now_add=True`). See https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.DateField.auto_now